frontend/src/Store/FiltersStore.ts

```diff
@@ -218,44 +213,40 @@ export class FiltersStore {
 
   // Resets all filters to their initial values.
   resetAllFilters() {
-    this._filterValues = { ...this._initialFilterValues };
-    this._rootStore.updateFilteredData();
+    this._rootStore.actions.resetAllFilters();
   }
```

Comment:
This kind of change is suggesting we should just flatten the stores into one, right?